src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -1146,30 +1024,11 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
                     // text is allowed to get the half of the available space rest is for buttons and version label
                     rect.width = (position.width - rect.x) / 2;
 
-                    EditorStyles.label.fontStyle = FontStyle.Bold;
-                    EditorStyles.label.fontSize = 16;
+                    var labelStyle = new GUIStyle(EditorStyles.label) { fontStyle = FontStyle.Bold, fontSize = 15 };
```

Comment:
Since this method is called in a loop for each package you should probably cache this style in a field.

Reply:
Like other styles, I cached it in the Styles class.